crate/operator/constants.py

```diff
@@ -28,7 +28,7 @@
 
 SYSTEM_USERNAME = "system"
 
-BACKOFF_TIME = 60.0
+CONNECT_TIMEOUT = 10.0
```

Comment:
Aren't these somewhat different concepts? `CONNECT_TIMEOUT` would suggest to me that it's simply a "fail if not connected by this time" rather than a "delay reconnect by this time"?

Reply:
Right, that's why I changed the name of the constant to CONNECT_TIMEOUT, and it is only used as argument of the `get_connection()` method https://github.com/crate/crate-operator/pull/137/files#diff-1af5aaa975261c7031681203924cb7afde32559b890a02d25a43cbe4531d8c93R283